pbl

```diff
@@ -72,6 +73,26 @@ Options:
   exit shift;
 }
 
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+# update_bootloader_usage($exit_code)
+#
+# Print help text and exit.
+#
+sub update_bootloader_usage
+{
+  print <<"= = = = = = = =";
+Usage: update_bootloader
+       bootloader_entry
+Configure/install boot loader.
+
+Options:
+    --help                  Write this help text.
```

Comment:
also `--reinit` is supported see https://github.com/openSUSE/perl-bootloader/pull/124/files#diff-25f66d09583995c8fbf858e439e0ab33R312

Reply:
Thanks, good catch